src/cr/cube/cubepart.py

```diff
@@ -1533,18 +1518,6 @@ def unweighted_counts(self):
         """1D np.int64 ndarray of unweighted count for each row of stripe."""
         return self._assembler.unweighted_counts
 
-    @lazyproperty
-    def var_scale_mean(self):
-        if np.all(np.isnan(self._numeric_values)):
-            return None
-
-        numeric_values = self._numeric_values[self._numeric_values_mask]
-        counts = self._counts_as_array[self._numeric_values_mask]
-
-        return np.nansum(counts * pow((numeric_values - self.scale_mean), 2)) / np.sum(
-            counts
-        )
-
```

Comment:
why get rid of this property?

Reply:
This property is used only by the `_ScaledCounts` measure and has moved to that class as `._scale_variance`. There were no clients for this so there is no interface property for it in `_Strand`. If we end up with a client for this then we can add `_Strand.scale_variance` and implement it as `return self._assembler.scale_variance` which in turn would call `self._measures.scaled_counts.scale_variance`.

As far as I can tell, this method should not have been public. Also, it was a smell that we needed helper methods like this for scaled-count computations rather than delegating that to it's own object. But all that is resolved by this measure-consolidation work.